PSXPackager/Program.cs

```diff
@@ -173,10 +170,9 @@ static void Main(string[] args)
 
         private static IEnumerable<string> GetFilesFromDirectory(string path, string filterExpression, bool recursive)
         {
+            // https://github.com/adamhathcock/sharpcompress/blob/master/FORMATS.md
             var supportedFiles = new List<string>() {
-#if SEVENZIP
-            ".7z", ".gz", ".rar", ".tar", 
-#endif
+            "rar", "zip", "tar", ".gz", ".7z",
```

Comment:
.rar, .zip, .tar

Reply:
I can fix this after I merge it in

Reply:
Thank you for fixing! https://github.com/RupertAvery/PSXPackager/commit/a08fe50e702e30af9acf3a3ae20b5ce17d7575f2